app/components/hearings-list-for-milestones-list.js

```diff
@@ -0,0 +1,66 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default class ProjectMilestoneComponent extends Component {
+  // @argument
+  milestone;
+
+  milestoneParticipantReviewLookup = {
+    BP: 'Borough President Review',
+    BB: 'Borough Board Review',
+    CB: 'Community Board Review',
+  }
+
+  // An array of disposition models that match the current milestone
+  @computed('milestone', 'milestone.project.dispositions')
+  get currentMilestoneDispositions() {
+    const currentMilestoneDispositions = [];
+
+    const milestone = this.get('milestone');
+    // ALL dispositions associated with a milestone's project
+    const dispositions = milestone.get('project.dispositions');
+    const milestoneParticipantReviewLookup = this.get('milestoneParticipantReviewLookup');
+
+    // iterate through ALL of the current project's dispositions
+    // IF a single disposition's assignment's dcpLupteammemberrole matches the
+    // current milestone's displayName, push that disposition into the new
+    // currentMilestoneDispositions array
+    dispositions.forEach(function(disposition) {
+      const participantType = disposition.get('assignment.dcpLupteammemberrole');
+
+      // milestoneParticipantReviewLookup[participantType] = e.g. `Community Board Review`
+      if (milestone.displayName === milestoneParticipantReviewLookup[participantType]) {
+        currentMilestoneDispositions.push(disposition);
+      }
+    });
+
+    // if dispositions have been pushed to the array (its length is greater than 1)
+    // return arrray -- otherwise return null
+    if (currentMilestoneDispositions.length < 1) {
+      return null;
+    } return currentMilestoneDispositions;
+  }
+
+  // An array of disposition models that match the current milestone
```

Comment:
Is this comment still true?

Reply:
Looks like it got shuffled around and is now in the wrong place -- will remove